code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -666,7 +679,6 @@ nobiliumsuppression = INFINITY
 /datum/gas_reaction/stimformation/init_reqs()
 	min_requirements = list(
 		/datum/gas/tritium = 30,
-		/datum/gas/plasma = 10,
 		/datum/gas/bz = 20,
 		/datum/gas/nitryl = 30,
 		"TEMP" = STIMULUM_HEAT_SCALE/2)
```

Comment:
pls explain, I assume this has to do with conflicting reactions, but I'm not sure

Reply:
is because in the future it may lead to unwanted reactions (already trit and bz makes another gas, but is at low temps) and also because one less gas may make the stim more appealing to make

Reply:
aight just make sure you mention it all in the pr desc

Reply:
yeah i'll write more in the description tomorrow

Reply:
ah you already did, I'm dumb